addon/components/au-toaster.hbs

```diff
@@ -1,17 +1,12 @@
 {{#if this.toaster.toasts.length}}
   <div class="au-c-toaster {{this.position}}">
     {{#each this.toaster.toasts as |toast|}}
-      <AuAlert
-        @title={{toast.title}}
-        @skin={{toast.options.type}}
-        @icon={{toast.options.icon}}
-        @size="small"
-        @closable={{toast.options.closable}}
-      >
-        {{#if toast.message}}
-          <p>{{toast.message}}</p>
-        {{/if}}
-      </AuAlert>
+      {{#let
+        (component (ensure-safe-component toast.name) title=toast.title message=toast.message options=toast.options)
```

Comment:
As mentioned in the other PR, `component` and `ensure-safe-component` shouldn't be needed here.

I went with passing only the options (and a close action) into the component since the `show` method doesn't accept a message and title and users have to pass everything through options anyway.

Reply:
Also see https://github.com/appuniversum/ember-appuniversum/pull/419#issuecomment-1680367652.

> I went with passing only the options (and a close action) into the component since the `show` method doesn't accept a message and title and users have to pass everything through options anyway.

Seems a tad more logical that way 👍 Glad to be learning!